src/stockTrader.js

```diff
@@ -41,6 +47,9 @@ export const StockTrader = () => {
           <TopPeers height="15%" />
         </DisplayWrapper>
       </DisplayWrapper>
+      <DisplayWrapper mt="2.0rem" width="100%" variant="flexRow">
```

Comment:
Do we need flexRow if there is only one thing in this row? Is width: 100% sufficient?

Reply:
You're right, don't need flexRow there although we might if we also add the favorites section. I removed it for now.